lib/peeps_repository.rb

```diff
@@ -0,0 +1,85 @@
+require_relative 'peeps'
+
+class PeepsRepository
+
+  # Selecting all records
+  # No arguments
+  def all
+    peeps = []
+    sql = 'SELECT id, message, time_of_post, user_id FROM peeps;'
+    result_set = DatabaseConnection.exec_params(sql, [])
+
+    result_set.each do |record|
+      peep = Peeps.new
+
+      peep.id = record['id']
+      peep.message = record['message']
+      peep.time_of_post = record['time_of_post']
+      peep.user_id = record['user_id']
+
+      peeps << peep
+    end
+        # Returns an array of Peep objects.
+    return peeps
+  end
+
+  def display_all_reverse
+    peeps = []
+    sql = 'SELECT id, message, time_of_post, user_id FROM peeps ORDER BY time_of_post DESC;'
+    result_set = DatabaseConnection.exec_params(sql, [])
+
+    result_set.each do |record|
+      peep = Peeps.new
+
+      peep.id = record['id']
+      peep.message = record['message']
+      peep.time_of_post = record['time_of_post']
+      peep.user_id = record['user_id']
+
+      peeps << peep
+    end
+        # Returns an array of Peep objects.
+    return peeps
+  end
```

Comment:
Cool use of a specific method to display in reverse, rather than simply altering the results of the '.all' method